### DISCUSSION TITLE
NI Relay Card Control

### DISCUSSION THREAD

**Discussion by osamaabualsoud** [ni/nidaqmx-python#572] [Q&A]
Hello, I want to control an NI 9482 relay card. Is there a specific function best suited for this task?

**Comment by zhindes** [comment]
The NI 9482 is represented in DAQmx as a Digital Output device. It should have 4 lines named, I think, "port0/line0" through "port0/line3". You can use those in Digital Output tasks. [Digital Output examples](https://github.com/ni/nidaqmx-python/tree/master/examples/digital_out) will give you a starting point. There are more examples being developed soon.
